### PR TITLE
return None instead of 0 in GetStdHandle

### PR DESCRIPTION
**Pull request by slozier**
Resolves an issue reported on gitter by @danabr:
> Hi. I have a mysterious issue with subprocess.Popen with stdout=subprocess.PIPE. It works fine from the IronPython console, and also when I run IronPython2 embedded in my application. However, when I use embedded IronPython3, I get: `OSError: [WinError 6] The handle is invalid`. Here is a stacktrace:
>
> ```
> File "myscript.py", line 137, in main
>     p = subprocess.Popen(commands, shell=True, stdout=subprocess.PIPE)
>   File "lib\subprocess.py", line 833, in __init__
>     (p2cread, p2cwrite,
>   File "lib\subprocess.py", line 1033, in _get_handles
>     p2cread = self._make_inheritable(p2cread)
>   File "lib\subprocess.py", line 1080, in _make_inheritable
>     h = _winapi.DuplicateHandle(
> ```
>
>The issue only occurs when I the stdin argument is not supplied or is set to None. The code in subprocess.py then tries to get the default stdin handle with _winapi.GetStdHandle(_winapi.STD_INPUT_HANDLE). That call returns 0 in this case.
>
>The code then goes on to compare the result with None, which seems wrong. From the docs of GetStdHandle:
>
>>    If an application does not have associated standard handles, such as a service running on an interactive desktop, and has not redirected them, the return value is NULL.

### CODE REVIEW

File: Tests/modules/network_related/test__ssl.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from iptest import IronPythonTestCase, is_cli, is_netcoreapp, retryOnFailure, run_test, skipUnlessIronPython
 
 SSL_URL      = "www.python.org"
-SSL_ISSUER   = "CN=GlobalSign Atlas R3 DV TLS CA 2022 Q3, O=GlobalSign nv-sa, C=BE"
```
Comment (slide) [inline Tests/modules/network_related/test__ssl.py:18]: Is this change really needed for the _winapi stuff?

Comment (slide) [reply Tests/modules/network_related/test__ssl.py:18]: nm, I see now

Comment (slozier) [reply Tests/modules/network_related/test__ssl.py:18]: I just wanted the green checkmark so I snuck it in. :smile:
